### ISSUE TITLE
Make instructions on usage in the browser clearer

### ISSUE THREAD

**Issue by oelna** [paulmillr/noble-bls12-381#30]
I have been spending some time with this already, but I can't get it to work. I can't be the only one.

The readme has basic usage instructions for Node.js and browser, but it is never explained how to include the Javascipt files in your project. I have tried

- adding `index.js` and `math.js` via `<script>` tag
- using ESM syntax with `import from`
- letting Browserify create a compiled javascript file

None of it worked. I'd have given up, but it does work on [NPM Runkit](https://npm.runkit.com/noble-bls12-381), so I *know* there is a way to do it, which is frustrating. I tried looking for a CDN version of the code, but it seems it does not yet exist either.

I don't know whether it makes sense to provide a compiled, single-file release version for the browser, but it sure would have made including this on a website easier.

**Comment by oelna** [comment]
Yes, that would be very helpful I think. I have built my own version of it in the meantime:
https://github.com/oelna/signed-qr-codes/blob/main/noble-bls.js

It works as an ES6 module in the browser (can be used via `import { nobleBLS as bls } from './noble-bls.js';`)

Disclaimer: I know nothing about any of this! I fiddled and deleted until something worked, but this should really be done by someone who knows what they are doing.
